dr/firmware/ucf/constraints.xdc

```diff
@@ -0,0 +1,2 @@
+create_clock -period [expr {round(1.0e6 / 360.0) / 1.0e3}] [get_ports clk_p]
+set_property HD.CLK_SRC BUFGCTRL_X0Y0 [get_ports clk_p]
```

Comment:
I like the idea, please move that to the top modules since all other modules would like to use the same file.